README.adoc

```diff
@@ -12,8 +12,8 @@
     . Add your SQL into resources directory file name: sample_sql.sql.
 Sometimes we need to hardcode some values in SQL, for that we can use syntax like:
         ```sql
-        SELECT * FROM table WHERE country = 'HARDCODE_AS_STRING{India}'
-        and customer_id = 'HARDCODE_AS_{300}'
+        SELECT * FROM table WHERE country = 'HARDCODE_AS_STRING\{India}'
```

Comment:
That backslash in `\{India}` is a escape in asciidoc. It will actually be rendered as `{India}`. If we don't put that backslash, `{India}` is interpreted as a variable.
See: https://docs.asciidoctor.org/asciidoc/latest/subs/prevent/#escape-with-backslashes